src/data-visualisation/accessibility/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: collection
+title: "Data visualisation accessibility"
```

Comment:
If this is a11y for datavis only, then should we put it under that section?
I think we need a top level a11y section, but the title indicates datavis only